packages/firestore/test/unit/local/persistence_test_helpers.ts

```diff
@@ -113,32 +113,29 @@ export async function testIndexedDbPersistence(
     await SimpleDb.delete(prefix + IndexedDbPersistence.MAIN_DATABASE);
   }
   const platform = PlatformSupport.getPlatform();
-  const persistence = IndexedDbPersistence.createIndexedDbPersistence({
-    allowTabSynchronization: !!options.synchronizeTabs,
-    persistenceKey: TEST_PERSISTENCE_PREFIX,
+  const persistence = new IndexedDbPersistence(
+    !!options.synchronizeTabs,
```

Comment:
I did not add the inline argument name here since I deemed `synchronizeTabs` more descriptive than `multiTab`. As stated above, both are probably descriptive enough on their own.